### ISSUE TITLE
ForbiddenGlobals exclude file not working

### ISSUE THREAD

**Issue by zdenekgebauer** [nunomaduro/phpinsights#240]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| Library version  | 1.7.0

My configurations file contains:
```php
    'config' => [
        \SlevomatCodingStandard\Sniffs\Variables\UnusedVariableSniff::class => [
            'exclude' => [
                'index.php'
            ],
        ],
        \NunoMaduro\PhpInsights\Domain\Insights\ForbiddenGlobals::class => [
            'exclude' => [
                'index.php'
            ],
        ],
    ]
```
and index.php contains unused local variable and  $_SERVER['HTTP_HOST'], $_SERVER['HTTP_X_REQUESTED_WITH'] . UnusedVariableSniff is ignored as expected, but  ForbiddenGlobals still show "globals accesses detected"

**Comment by olivernybroe** [comment]
All insights which are not sniffs does not support the `exclude` parameter. 
It is definitely something we need.

**Comment by Gummibeer** [comment]
@olivernybroe this should stated clearer or get fixed.

> You can also configure the exclude parameter on each insight, to disallow an insight on a specific file.

https://phpinsights.com/insights/#configure-insights-2

This sentence, from the docs, is the exact opposit of what you wrote and seems to be the case.

**Comment by olivernybroe** [comment]
@gummibeer you are right, this is definitely missing in the docs. 

It might make sense the fix the docs and then when we have actually fixed the other insights to have this support, we can change the docs again.

**Comment by Gummibeer** [comment]
Will do so in combination with #237

**Comment by Jibbarth** [comment]
Fixed in 1.10 :+1:
